alerts/cluster-monitoring-operator/KubeDaemonSetMisScheduled.md

```diff
@@ -0,0 +1,68 @@
+# KubeDaemonSetMisScheduled
+
+## Meaning
+
+The `KubeDaemonSetMisScheduled` alert triggers when, over a defined time interval, an excessive number of daemonset pods are found running on nodes where they should not be. This typically occurs when a pod is scheduled on a node with a taint that doesn't have a corresponding toleration, or when a node is unhealthy or overloading, hindering pod scheduling.
+
+## Impact
+
+Excessive resource usage where they could be used by other applications
+
+## Diagnosis
+
+- Check Daemonset status
+
+  ```console
+  oc -n $NAMESPACE describe daemonset $NAME
+  ```
+
+- Check the status of the pods belonging to the Daemonset and look for the fields Node, Tolerations and NodeSelector:
+
+  ```console
+  oc -n $NAMESPACE describe pod $NAME
+  ```
+
+  Replace $NAME with the name of the pod having issue.
+
+  -  Look for the `Node`, `Node-Selectors` and `Tolerations` fields in the output
+
+- Check labels for the node and verify with `NodeSelector` on Pod
+
+  ```console
+  oc get node $NAME --show-labels
+  ```
+
+  Replace $NAME with the name of the pod having issue.
+
+    - Verify whether `Node-Selector` field for pod has one of the labels of node
+
+- Check Taint for the node and verify with `Toleration` on Pod
```

Comment:
```suggestion
- Check the `Taint` for the node and verify that it matches the `Toleration` on the pod.
```
Not sure if this rewording is till accurate, but the current wording is unclear to me.